Document autodiscover dedot option (#10938)
elastic/beats: filebeat/docs/autodiscover-docker-config.asciidoc

-------------------------------------------------------------------------------------

This configuration launches a `docker` logs input for all containers running an image with `redis` in the name.

> `labels.dedot` is default to be `true` for docker autodiscover, which means dots in docker labels are replaced with '_' by default.

Reviewer: ```suggestion
`labels.dedot` defaults to `true` for docker autodiscover, which means that dots in docker labels are replaced with '_' by default.
```
